diff --git a/lib/publish.js b/lib/publish.js
index ff3af00789942572806e52627b5e22c8ac25d1b1..8b48be405eefdeb51bdbcc837b4b88d35253dd5b 100644
--- a/lib/publish.js
+++ b/lib/publish.js
@@ -20,10 +20,11 @@ module.exports = async (npmrc, {npmPublish, pkgRoot}, pkg, context) => {
     const distTag = getChannel(channel);
 
     logger.log(`Publishing version ${version} to npm registry on dist-tag ${distTag}`);
+    logger.log('Using patched version of npm publish');
     const result = execa(
       'npm',
-      ['publish', basePath, '--userconfig', npmrc, '--tag', distTag, '--registry', registry],
-      {cwd, env, preferLocal: true}
+      ['publish', '--userconfig', npmrc, '--tag', distTag, '--registry', registry],
+      {cwd: basePath, env, preferLocal: true}
     );
     result.stdout.pipe(stdout, {end: false});
     result.stderr.pipe(stderr, {end: false});
